server/utils/cloudFunctions/functions/firebaseConfig.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Description: Initialize the index environment
+ *              Basically prepare firestore and algolia
```

Comment:
nit: Lots of "basically"s in your comments. I think it's a little unsure-sounding/informal